crates/iceberg-catalog/src/api/management/v1/warehouse.rs

```diff
@@ -26,29 +31,97 @@ pub struct CreateWarehouseResponse {
     pub warehouse_id: uuid::Uuid,
 }
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
+#[serde(rename_all = "kebab-case")]
+pub struct UpdateWarehouseStorageRequest {
+    /// Storage profile to use for the warehouse.
+    pub storage_profile: StorageProfile,
+    /// Optional storage credential to use for the warehouse.
+    /// The existing credential is not re-used. If no credential is
+    /// provided, we assume that this storage does not require credentials.
+    pub storage_credential: Option<StorageCredential>,
+}
+
+#[derive(Debug, Deserialize, ToSchema)]
+#[serde(rename_all = "kebab-case")]
+pub struct ListWarehouseRequest {
+    /// Optional filter to include inactive warehouses.
+    #[serde(default)]
+    pub include_inactive: Option<bool>,
```

Comment:
I this an Option to please the ToSchema gods to make it appear as not required in the openapi spec? If not then we could consider omitting the option, bool::default() should give false which I assume is what None also means.

Reply:
Exactly, it's meant for openapi. I will have to check how it shows up there :D